src/platform-includes/getting-started-primer/apple.mdx

```diff
@@ -38,3 +38,4 @@
 - [Attachments](/platforms/apple/enriching-events/attachments/) enrich your event by storing additional files, such as config or log files
 - [User Feedback](/platforms/apple/enriching-events/user-feedback/) provides the ability to collect user information when an event occurs
 - [Screenshot attachments for errors](/platforms/apple/guides/ios/enriching-events/screenshots/)
+- [Source Context](/platforms/apple/data-management/debug-files/source-context/) shows snippets of code around the location of stack frames
```

Comment:
Is this being added twice?

Reply:
Where is the other place?

Reply:
Not in here 
![image](https://github.com/getsentry/sentry-docs/assets/28265868/3db3be8f-2e0c-4345-b037-9e1a836781a8)

Reply:
there are multiple primer files and I've mistaken it was a single one, good catch.